### DISCUSSION TITLE
Azure Lighthouse & Domain-joined VMs

### DISCUSSION THREAD

**Discussion by PiotrWachulec** [AzureCommunityPL/group#292] [Pomoc]
Cześć ;) Case jest taki. Użytkownicy z tenanta A mają dostęp do zasobów w tenancie B poprzez konfigurację Azure Lighthouse pomiędzy tenantami. Użytkownicy z tenanta A potrzebują dostać się do wirtualnych maszyn podłączonych do domeny w tenancie B  po protokole RDP. Pytanie jest następujące - czy istnieje jakakolwiek inna możliwość dania im owego dostępu do VM-ek niż stworzenie dla nich kont w domenie w tenancie B / utworzenie lokalnych kont administracyjnych?

**Comment by sbugalski** [comment]
Cześć,

To może być częściowe rozwiązanie Twojego problemu. Tylko jak pamiętam to Guest users nie mogli się logowac, nie moge odnaleźć info czy coś się zmieniło od ostatniego razu.

**Reply by mgajownik** [reply]
nie zmieniło się :( Azure B2B nie daje możliwości takiej, ale może extarnal user flow coś pomoże
https://docs.microsoft.com/en-us/azure/active-directory/external-identities/self-service-sign-up-user-flow

**Reply by PiotrWachulec** [reply]
Rozumiem :( Dzięki Panowie za odpowiedzi! :)
